.github/workflows/workflow.yaml

```diff
@@ -64,7 +75,7 @@ jobs:
           kubectl logs -n gatekeeper-system -l control-plane=audit-controller --tail=-1 > logs-audit.json
 
       - name: Upload artifacts
-        uses: actions/upload-artifact@v2
+        uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab
```

Comment:
@nilekhc Just wanted to make sure this change was intentional?

Reply:
It is. It's always better to pin to the sha version. I have also opened https://github.com/open-policy-agent/gatekeeper-library/pull/322 to automatically update it.

Reply:
@nilekhc Sorry, to clarify I meant the changing action from `upload-artifact` to `checkout`?  No concerns on using the SHA instead of tag, that is very common.

Reply:
Updated. Thanks.